src/components/search/no-results-message.vue

```diff
@@ -1,22 +1,27 @@
 <template>
   <i18n
     v-if="$x.noResults"
-    class="x-no-results-message x-text1 x-flex x-flex-col x-items-center x-gap-8 x-break-words x-bg-neutral-10 x-p-24 x-text-center desktop:x-text1-lg"
+    class="x-no-results-message x-text1 x-mt-24 x-flex x-items-center x-break-words x-bg-neutral-10 x-py-8 x-text-center x-text-neutral-90 desktop:x-flex-wrap desktop:x-justify-center desktop:x-gap-8 desktop:x-text1-lg desktop:x-py-24"
```

Comment:
Why are we changing so many text colors from `x-text1` to `x-text-neutral-90`?

Reply:
Many texts need the color `x-text-neutral-90`, I mean this color `#283034`, but by default, all these texts have associated this color `#0000`. We could separate these text changes in another task and investigate how to fix them.